scripts/windows_live_validate.ps1

```
@@ -17,12 +17,28 @@ Set-StrictMode -Version Latest
 $ConfigPath = Join-Path $RepoRoot 'config/windows-parakeet.toml'
-$PluginConfigPath = Join-Path $ArtifactRoot 'plugins.json'
 $LogPath = Join-Path $RepoRoot 'logs/coldvox.log'
-$SmokeFeatureList = 'silero,text-injection-enigo'
-$LiveFeatureList = 'parakeet,silero,text-injection-enigo'
-$ParakeetDevice = if ($env:PARAKEET_DEVICE) { $env:PARAKEET_DEVICE } else { 'cuda' }
+$TestWavPath = Join-Path $RepoRoot 'crates/app/test_data/test_1.wav'
+$HealthUrl = 'http://localhost:5092/health'
+$TranscriptionsUrl = 'http://localhost:5092/v1/audio/transcriptions'
+$FeatureList = 'http-remote,text-injection-enigo'
+
+function Get-ParakeetHealthTimeoutSeconds {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Get-ParakeetHealthTimeoutSeconds' uses a plural noun. A singular noun should be used instead.

Suggested fix: Singularized correction of 'Get-ParakeetHealthTimeoutSeconds'

(PSUseSingularNouns)